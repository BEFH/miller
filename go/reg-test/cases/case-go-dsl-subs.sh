echo x=eeee | run_mlr put '$y=ssub($x, "e", "X")'
echo x=eeee | run_mlr put '$y=sub($x, "e", "X")'
echo x=eeee | run_mlr put '$y=gsub($x, "e", "X")'
