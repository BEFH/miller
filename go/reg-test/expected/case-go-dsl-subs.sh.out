
mlr put $y=ssub($x, "e", "X")
x=eeee,y=Xeee

mlr put $y=sub($x, "e", "X")
x=eeee,y=Xeee

mlr put $y=gsub($x, "e", "X")
x=eeee,y=XXXX

